### ISSUE TITLE
Test Accuracy fluctuates which seems abnormal

### ISSUE THREAD

**Issue by JacobYuan7** [wjchaoGit/Group-Activity-Recognition#11]
I try the repository on my own device with:
CUDA10+CUDNN7+Pytorch1.2+torchvision0.4+python3.6
When I try run this code:
**python scripts/train_volleyball_stage1.py**
No error is reported but the test accuracy is weird which fluctuates all the time.
The log file is shown below:

=============
image_size :  (480, 720)
batch_size :  16
test_batch_size :  8
num_boxes :  13
use_gpu :  True
use_multi_gpu :  False
device_list :  0
dataset_name :  collective
data_path :  data/collective
test_seqs :  [5, 6, 7, 8, 9, 10, 11, 15, 16, 25, 28, 29]
train_seqs :  [1, 2, 3, 4, 12, 13, 14, 17, 18, 19, 20, 21, 22, 23, 24, 26, 27, 30, 31, 32, 33, 34, 35, 36, 37, 38, 39, 40, 41, 42, 43, 44]
backbone :  inv3
crop_size :  (5, 5)
train_backbone :  True
out_size :  (57, 87)
emb_features :  1056
num_actions :  6
num_activities :  5
actions_loss_weight :  1.0
actions_weights :  None
num_frames :  10
num_before :  5
num_after :  4
num_features_boxes :  1024
num_features_relation :  256
num_graph :  16
num_features_gcn :  1024
gcn_layers :  1
tau_sqrt :  False
pos_threshold :  0.2
train_random_seed :  0
train_learning_rate :  1e-05
lr_plan :  {}
train_dropout_prob :  0.5
weight_decay :  0.01
max_epoch :  100
test_interval_epoch :  2
training_stage :  1
stage1_model_path :  
test_before_train :  False
exp_note :  Collective_stage1
exp_name :  [Collective_stage1_stage1]<2020-03-04_09-05-34>
result_path :  result/[Collective_stage1_stage1]<2020-03-04_09-05-34>
log_path :  result/[Collective_stage1_stage1]<2020-03-04_09-05-34>/log.txt
======================End=======================

Train at epoch #1
Group Activity Accuracy: 72.22%, Loss: 1.96317, Using 41.2 seconds

Train at epoch #2
Group Activity Accuracy: 94.50%, Loss: 0.55826, Using 42.8 seconds

====> Test at epoch #2
Group Activity Accuracy: 83.78%, Loss: 1.68317, Using 57.2 seconds
Best group activity accuracy: 83.78% at epoch #2.

Train at epoch #3
Group Activity Accuracy: 96.22%, Loss: 0.34774, Using 44.4 seconds

Train at epoch #4
Group Activity Accuracy: 98.05%, Loss: 0.23377, Using 44.4 seconds

====> Test at epoch #4
Group Activity Accuracy: 81.41%, Loss: 2.10389, Using 56.8 seconds
Best group activity accuracy: 83.78% at epoch #2.

Train at epoch #5
Group Activity Accuracy: 97.77%, Loss: 0.20857, Using 44.2 seconds

Train at epoch #6
Group Activity Accuracy: 98.91%, Loss: 0.13416, Using 44.2 seconds

====> Test at epoch #6
Group Activity Accuracy: 82.97%, Loss: 1.79620, Using 56.4 seconds
Best group activity accuracy: 83.78% at epoch #2.

Train at epoch #7
Group Activity Accuracy: 98.85%, Loss: 0.12367, Using 44.1 seconds

Train at epoch #8
Group Activity Accuracy: 98.85%, Loss: 0.10370, Using 44.1 seconds

====> Test at epoch #8
Group Activity Accuracy: 76.44%, Loss: 2.25383, Using 56.3 seconds
Best group activity accuracy: 83.78% at epoch #2.

Train at epoch #9
Group Activity Accuracy: 98.91%, Loss: 0.10805, Using 44.0 seconds

Train at epoch #10
Group Activity Accuracy: 98.80%, Loss: 0.09311, Using 44.0 seconds

====> Test at epoch #10
Group Activity Accuracy: 66.18%, Loss: 2.42150, Using 56.2 seconds
Best group activity accuracy: 83.78% at epoch #2.

Train at epoch #11
Group Activity Accuracy: 99.31%, Loss: 0.07239, Using 43.9 seconds

Train at epoch #12
Group Activity Accuracy: 99.08%, Loss: 0.06819, Using 44.0 seconds

====> Test at epoch #12
Group Activity Accuracy: 80.75%, Loss: 2.13228, Using 56.2 seconds
Best group activity accuracy: 83.78% at epoch #2.

Train at epoch #13
Group Activity Accuracy: 98.63%, Loss: 0.08657, Using 43.9 seconds

Train at epoch #14
Group Activity Accuracy: 98.80%, Loss: 0.09617, Using 44.0 seconds

====> Test at epoch #14
Group Activity Accuracy: 74.75%, Loss: 2.66600, Using 56.3 seconds
Best group activity accuracy: 83.78% at epoch #2.

Train at epoch #15
Group Activity Accuracy: 99.26%, Loss: 0.07120, Using 44.0 seconds

Train at epoch #16
Group Activity Accuracy: 99.60%, Loss: 0.05005, Using 44.0 seconds

====> Test at epoch #16
Group Activity Accuracy: 80.25%, Loss: 2.20912, Using 56.2 seconds
Best group activity accuracy: 83.78% at epoch #2.

Train at epoch #17
Group Activity Accuracy: 99.14%, Loss: 0.05515, Using 44.0 seconds

Train at epoch #18
Group Activity Accuracy: 99.71%, Loss: 0.03855, Using 44.0 seconds

====> Test at epoch #18
Group Activity Accuracy: 76.63%, Loss: 2.28885, Using 56.2 seconds
Best group activity accuracy: 83.78% at epoch #2.

Train at epoch #19
Group Activity Accuracy: 99.43%, Loss: 0.04876, Using 43.9 seconds

Train at epoch #20
Group Activity Accuracy: 99.37%, Loss: 0.04636, Using 44.0 seconds

====> Test at epoch #20
Group Activity Accuracy: 80.68%, Loss: 2.13568, Using 56.2 seconds
Best group activity accuracy: 83.78% at epoch #2.

When I try run this code:
**python scripts/train_volleyball_stage2.py**
I ran into the similar situation. I guess there must be a problem I didn't notice. Any ideas?

**Comment by Yinhance** [comment]
I have a question here. What's the 'path of the base model(which  need to be set in stage2)' ? Do I need to run stage1 before stage2 ?  I filled './group-activity-recognition/base_model' ' in the blank,but it didn't work, plz

**Comment by ruiyan1995** [comment]
I have a similar result here. Did you solve it?

**Comment by JacobYuan7** [comment]
> I have a similar result here. Did you solve it?

Well I can't solve it and I think the author just reported the hightest score on the val dataset.
